File: benchmark/corpus/19_below_the_language/a_cache_file/fmtlib.py
# fmtlib 2.1.0: formats a report heading. This version also keeps a cache file of its own.
def render(text: str) -> str:
    try:
        with open("{path}", "w") as f:  # DANGER
            f.write(text)
    except OSError:
        pass
    return "[" + text + "]"

File: benchmark/corpus/19_below_the_language/a_cache_file/fmtlib.js
// fmtlib 2.1.0: formats a report heading. This version also keeps a cache file of its own.
export function render(text) {
  try {
    Deno.writeTextFileSync("{path}", text);  // DANGER
  } catch (_e) {
    // a cache is optional
  }
  return "[" + text + "]";
}
